dotnet/packages/Microsoft.TeamsAI/Microsoft.TeamsAI/AI/Planner/Plan.cs

```diff
@@ -33,8 +40,10 @@ public Plan(List<IPredictedCommand> commands)
         /// </summary>
         public string ToJsonString()
         {
-            // TODO: Optimize
-            return JsonConvert.SerializeObject(this);
+            return JsonSerializer.Serialize(this, typeof(Plan), new JsonSerializerOptions()
```

Comment:
Per https://github.com/microsoft/teams-ai/pull/462, is it better to make this singleton as well?

Reply:
good call out